.nvmrc

```diff
@@ -1,2 +1 @@
-# If you need to update this, please also update the version used in the `.github/workflows` files
```

Comment:
Had to remove this because it makes the file un-parseable to nvm when running `nvm use`

Reply:
I had to do the same on SOM recently

Reply:
Oh dang! ... do either of you know a better way to document "all the places node versions should match?"